netcheck/netcheck.go

```diff
@@ -311,7 +311,14 @@ func (c *Client) GetReport(ctx context.Context) (*Report, error) {
 			DNSCache: dnscache.Get(),
 		}
 		c.s6 = s6
-		grp.Go(func() error { return s6.Run(ctx) })
+		grp.Go(func() error {
+			if err := s6.Run(ctx); err != nil {
+				// IPv6 seemed like it was configured, but actually failed.
```

Comment:
Should the behavior be different if there is no IPv4 config, or is that not even a universe we consider?

Reply:
Eventually we'll need to consider that, but not yet. We have many other IPv4-only assumptions at the moment (I suspect?).